spec/supabase_dart_v2.yml

```diff
@@ -0,0 +1,3024 @@
+          final data = await supabase
+            .from('cities')
+            .delete()
+            .lte('country_id', 250);
+          ```
+      - id: with-rpc
+        name: With `rpc()`
+        code: |
+          ```dart
+          // Only valid if the Stored Procedure returns a table type.
+          final data = await supabase
+            .rpc('echo_all_cities')
+            .lte('country_id', 250);
+          ```
+
+  - id: like
+    title: like()
+    description: |
+      Finds all rows whose value in the stated `column` matches the supplied `pattern` (case sensitive).
+    $ref: '@supabase/postgrest-js.PostgrestFilterBuilder.lte'
+    examples:
+      - id: with-select
+        name: With `select()`
+        isSpotlight: true
+        code: |
+          ```dart
+          final data = await supabase
+            .from('cities')
+            .select('name, country_id')
+            .like('name', '%la%');
+          ```
+      - id: with-update
+        name: With `update()`
+        code: |
+          ```dart
+          final data = await supabase
+            .from('cities')
+            .update({ 'name': 'Mordor' })
+            .like('name', '%la%');
+          ```
+      - id: with-delete
+        name: With `delete()`
+        code: |
+          ```dart
+          final data = await supabase
+            .from('cities')
+            .delete()
+            .like('name', '%la%');
+          ```
+      - id: with-rpc
+        name: With `rpc()`
+        code: |
+          ```dart
+          // Only valid if the Stored Procedure returns a table type.
+          final data = await supabase
+            .rpc('echo_all_cities')
+            .like('name', '%la%');
+          ```
+
+  - id: ilike
+    title: ilike()
+    description: |
+      Finds all rows whose value in the stated `column` matches the supplied `pattern` (case insensitive).
+    examples:
+      - id: with-select
+        name: With `select()`
+        isSpotlight: true
+        code: |
+          ```dart
+          final data = await supabase
+            .from('cities')
+            .select('name, country_id')
+            .ilike('name', '%la%');
+          ```
+      - id: with-update
+        name: With `update()`
+        code: |
+          ```dart
+          final data = await supabase
+            .from('cities')
+            .update({ 'name': 'Mordor' })
+            .ilike('name', '%la%');
+          ```
+      - id: with-delete
+        name: With `delete()`
+        code: |
+          ```dart
+          final data = await supabase
+            .from('cities')
+            .delete()
+            .ilike('name', '%la%');
+          ```
+      - id: with-rpc
+        name: With `rpc()`
+        code: |
+          ```dart
+          // Only valid if the Stored Procedure returns a table type.
+          final data = await supabase
+            .rpc('echo_all_cities')
+            .ilike('name', '%la%');
+          ```
+
+  - id: is
+    title: is_()
```

Comment:
I thought this was changed to `isFilter`, am I wrong?

Reply:
Sorry, not sure how this happened. Will fix it.